connections/README.md

```diff
@@ -112,8 +112,8 @@ functions.
 Each protocol supported by a peer is identified using a unique string called a
 **protocol id**. While any string can be used, the conventional format is a
 path-like structure containing a short name and a version number, separated by
-`/` characters. For example: `/mplex/1.0.0` identifies version 1.0.0 of the
-[`mplex` stream multiplexing protocol][mplex]. multistream-select itself has a
+`/` characters. For example: `/yamux/1.0.0` identifies version 1.0.0 of the
```

Comment:
That's not the version number we use for yamux.

Reply:
Are you sure @marten-seemann? It matches the one referenced in `go-libp2p/defaults.go`:

https://github.com/libp2p/go-libp2p/blob/228c6297685f2d95577680128df982e827abf694/defaults.go#L31-L38

Reply:
Oh sorry, I mixed this up with mplex!